src/QuickPad.UI/QuickPad.UI/Dialogs/AskToSave.xaml

```diff
@@ -13,6 +13,7 @@
     Foreground="{x:Bind converters:Converter.FromColorToBrush(VtSelector.CurrentItem.DefaultTextForegroundColor)}"
     PrimaryButtonCommand="{x:Bind Commands.SaveCommand}"
     PrimaryButtonCommandParameter="{x:Bind ViewModel}"
+    PrimaryButtonStyle="{ThemeResource AccentButtonStyle}"
```

Comment:
I am pretty sure this style includes rounded corners, we should probably add the standard style template to the secondary buttons as well.

Reply:
Sorry, I forgot to change the styling afterwards